### ISSUE TITLE
Dupplicated Polytopes.jl and PolytopesTests.jl files

### ISSUE THREAD

**Issue by fverdugo** [gridap/Gridap.jl#36]
@santiagobadia  I have just realized that we have duplicated files:
`src/RefFEs/Polytopes.jl`
`src/Geometry/Polytopes.jl`
on the one hand, and
`test/RefFEsTests/PolytopesTests.jl`
`test/PolytopesTests.jl`
on the other hand.

The ones that we want to keep are:  
`src/RefFEs/Polytopes.jl`
`test/RefFEsTests/PolytopesTests.jl`
They are the ones being used and tested in the library.

The other two:
`src/Geometry/Polytopes.jl`
`test/PolytopesTests.jl`
seem to be there by accident, they are not being used and need to be deleted.

Please, make sure that you are working on the good ones and that you have not relevant changes in the wrong ones.

Thanks!

**Comment by santiagobadia** [comment]
@fverdugo I realized that there were two versions after some time trying to understand why what I implemented was not accessible at the test file. I was working on the file not being included. Next, I passed all changes to the good one. I thought you were aware of this (I was not) and I wanted to ask you tomorrow why we have two versions of the file and whether it was possible to eliminate the one not being used.
